tasks/nexus_install.yml

```diff
@@ -18,6 +29,8 @@
     src: "{{ nexus_download_dir }}/{{ nexus_package }}"
     dest: "{{ nexus_installation_dir }}"
     creates: "{{ nexus_installation_dir }}/nexus-{{ nexus_version }}"
+    owner: "{{ nexus_os_user }}"
```

Comment:
Rolling back, had problems deploying scripts to the script dir located in nexus_install dir. never mind, i've found a simple solution.